docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -455,9 +458,8 @@ Once a testing label is added to the PR and the tests are run, the results of th
 
    *Sample of Jenkins Test Results*
 
-Once on the `Jenkins <https://jenkins-epic.woc.noaa.gov>`__ page specific to the PR check in question, users can view all of the testing output, including "artifacts" from the build. To do this, users must click on the arrow icon in the top right corner of the page. Then, in the left navigation menu, they can click on *S3 Artifacts* and download any files listed there for in-depth review.
-
-
+Once on the `Jenkins <https://jenkins-epic.woc.noaa.gov>`__ page specific to the PR check in question, users can view all of the testing output, including "artifacts" from the build. To do this, users must click on the arrow icon in the top right corner of the page. Then, in the lefthand navigation menu, they can click on *S3 Artifacts* and download any files listed there for in-depth review.
```

Comment:
I am no longer able to see the Jenkins runs -- greeted with a login page instead. Is this not the case for everybody?

Reply:
Jesse found a security issue with Jenkins related to allowing access to SRW jobs to the public.  All of the Jenkins usernames are exposed.  Jesse disabled access for the public to view jobs in Jenkins to ensure that this wouldn't become an issue.

It would be best to either remove this section or add a note that this functionality isn't currently available to the public.

Reply:
Deleted! :)